pkg/webui/lib/field-description-messages.js

```diff
@@ -0,0 +1,174 @@
+  joinServerAbsence:
+    'If the end device is pre-provisioned, you will need the keys from the manufacturer to activate it.',
+
+  devAddrDescription:
+    'A 32 bit non-unique identifier, assigned by the Network Server during end device activation.',
+  devAddrLocation:
+    'For OTAA, the address is created by the Network Server. If using ABP, you must create one and manually enter it in the end device and The Things Stack.',
+
+  appSKeyDescription:
+    'After activation, this encryption key is used to secure messages which carry a payload.',
+  appSKeyLocation:
+    'For OTAA, the key is generated by the Network Server. If using ABP, you must create one and manually enter it in the end device and The Things Stack.',
+
+  nwkSKeyDescription:
+    'After activation, this encryption key is used to secure messages which do not carry a payload.',
+  nwkSKeyLocation:
+    'For OTAA, it is created by the Network Server. If using ABP, you must create one and manually enter it in the end device and The Things Stack.',
+
+  lwVersionDescription:
+    'The LoRa Alliance LoRaWAN specification your end device conforms to, which defines which Media Access Control features it supports.',
+  lwVersionLocation:
+    'The LoRaWAN version for your end device should be provided by the manufacturer in a datasheet as LoRaWAN version or LoRaWAN specification',
+  lwVersionAbsence:
+    'Contact your manufacturer or reseller, since specifying the wrong version can lead to complex issues when the Network Server provides the end device with unsupported configuration (MAC) commands.',
+
+  regParameterDescription:
+    'The Regional Parameters specify frequency, dwell time, and other communication settings for different geographical areas. The Regional Parameters version is the version of the LoRa Alliance specification which your device supports.',
+  regParameterLocation:
+    'The Regional Parameters version should be provided by the end device manufacturer in a datasheet.',
+  regParameterAbsence:
+    'Contact your manufacturer or reseller, since specifying the wrong version can lead to complex issues when the Network Server provides the end device with unsupported configuration (MAC) commands.',
+
+  classDescription:
+    'The LoRaWAN specification defines three end device types. All LoRaWAN end devices must implement Class A, whereas Class B and Class C are extensions to the specification of Class A devices.',
+  classLocation:
+    'The class capabilities of your end device should be provided by the manufaturer in a datasheet.',
+  classAbsence:
+    'All end devices support Class A, so you can always use it. Contact your manufacturer or reseller if you can not find information about what additional classes your end device supports.',
```

Comment:
```suggestion
    'If your end device will not receive downlink messages, you can safely use Class A, which all LoRaWAN certified devices must implement. Otherwise contact your manufacturer or reseller.',
```